debian/postinst

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+update-alternatives --install /usr/bin/x-terminal-emulator x-terminal-emulator /usr/bin/alacritty 50
```

Comment:
I'm not sure if this should be done automatically. Installing a deb and suddenly having a different default terminal emulator seems a bit backwards to me.

I'd instead recommend to provide instructions how to do this in the INSTALL.md.